.changeset/hot-cats-happen.md

```diff
@@ -0,0 +1,5 @@
+---
+"@fuel-ts/account": patch
+---
+
+- Fix middleware exception when making a request to an offline provider
```

Comment:
Our changesets must follow the PR title. 🙂 

```suggestion
fix: error handling in `Provider` for when a node is offline
```

Reply:
I see, thank you!

Reply:
It needs the `fix: ` prefix. 🙃 

```
fix: error handling in `Provider` for when a node is offline
```